package.json

```diff
@@ -26,38 +26,43 @@
     "url": "https://github.com/meetup/meetup-web-components/issues"
   },
   "homepage": "https://github.com/meetup/meetup-web-components#readme",
-  "dependencies": {
+  "peerDependencies": {
+    "classnames": "^2.2.5",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-intl": "^2.2.3",
+    "react-router": "^3.0.2"
+  },
+  "devDependencies": {
+    "@kadira/storybook": "2.35.3",
     "babel-cli": "6.22.2",
     "babel-core": "6.22.0",
     "babel-jest": "18.0.0",
+    "babel-loader": "6.2.10",
     "babel-plugin-react-intl": "2.3.0",
     "babel-plugin-react-transform": "2.0.2",
     "babel-preset-es2015": "6.18.0",
     "babel-preset-react": "6.16.0",
     "babel-preset-stage-2": "6.18.0",
     "babel-register": "6.18.0",
     "classnames": "2.2.5",
+    "coveralls": "2.11.15",
+    "css-loader": "0.26.1",
     "eslint": "3.13.1",
+    "eslint-loader": "1.6.1",
     "eslint-plugin-react": "6.9.0",
-    "meetup-web-mocks": "0.1.41",
+    "inquirer": "2.0.0",
+    "jest-cli": "18.1.0",
+    "meetup-web-mocks": "0.1.51",
     "node-sass": "4.3.0",
+    "npm-run-all": "4.0.1",
     "react": "15.4.2",
     "react-addons-test-utils": "15.4.2",
     "react-dom": "15.4.2",
     "react-intl": "2.2.3",
-    "react-transform-catch-errors": "1.0.2",
-    "sassquatch2": "2.4.4"
-  },
-  "devDependencies": {
-    "@kadira/storybook": "2.35.3",
-    "babel-loader": "6.2.10",
-    "coveralls": "2.11.15",
-    "css-loader": "0.26.1",
-    "eslint-loader": "1.6.1",
-    "inquirer": "2.0.0",
-    "jest-cli": "18.1.0",
-    "npm-run-all": "4.0.1",
+    "react-router": "3.0.2",
     "sass-loader": "4.1.1",
+    "sassquatch2": "2.4.4",
```

Comment:
addressed. but it also needs a devDep reference for storybook. it appears to be common that peerDependencies need a devDependency version as well if they're used in testing.